src/cdk/v2/destinations/optimizely_fullstack/procWorkflow.yaml

```diff
@@ -172,17 +172,14 @@ steps:
         template: |
           const eventsMap = $.getHashFromArray(.destination.Config.eventMapping, 'from', 'to');
           const eventInLowerCase = .message.event.toLowerCase();
-          const mappedEvent = eventsMap[eventInLowerCase];
-          $.assert(mappedEvent, "No mapped event found for " + .message.event + 
-          " event. Make sure to map the event in 'RudderStack to Optimizely event mappings' setting");
-          mappedEvent;
+          const mappedEvent = eventsMap[eventInLowerCase] || eventInLowerCase;
```

Comment:
Is it okay to store the event name in lowercase? The customer can put the name in upper case as well in Optimizely. We shouldn't alter the case of the event.